charts/linkerd2/templates/web.yaml

```diff
@@ -63,6 +63,8 @@ spec:
         - -grafana-addr=linkerd-grafana.{{.Namespace}}.svc.{{.ClusterDomain}}:3000
         - -controller-namespace={{.Namespace}}
         - -log-level={{.ControllerLogLevel}}
+        {{- $host := replace "." "\\." (printf "linkerd-web.%s.svc.%s" .Namespace .ClusterDomain) }}
+        - -enforced-host=^(localhost|127\.0\.0\.1|{{ $host }}|\[::1\]):\d+$
```

Comment:
should the port be optional in this regex?

Reply:
Indeed!